src/main/scala/beam/agentsim/agents/ridehail/OutOfServiceVehicleManager.scala

```diff
@@ -49,13 +41,13 @@ class OutOfServiceVehicleManager(
       )
   }
 
-  def registerTrigger(vehicleId: Id[Vehicle], triggerId: Option[Long]) = {
+  def registerTrigger(vehicleId: Id[Vehicle], triggerId: Option[Long]): Option[Option[Long]] = {
```

Comment:
Option[Option[Long]] I think one level optional is enough, lets flatten it

Reply:
the only statement of the method is` triggerIds.put(vehicleId, triggerId)` (put Option[Long] to map) and it returns Option[VAL] that makes it Option[Option[Long]]. Please suggest the fix.

Reply:
`triggerIds.put(vehicleId, triggerId).flatten`